tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -188,15 +191,15 @@ public function iProceedToTheAddressingStep(): void
      */
     public function iProceedToTheShippingStep(): void
     {
-
+        $this->session->getPage()->find('css', '#next-step')->click();
```

Comment:
This behaviour should be in page.

And for example, [AddressPageInterface](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Page/Shop/Checkout/AddressPageInterface.php#L93) has method `nextStep` which you can use here